geoportailv3/static/js/catalog/layertreenode.html

```diff
@@ -1,14 +1,14 @@
 <a data-toggle="collapse" data-parent="#catalog-{{::layertreenodeCtrl.parentUid}}"
    href="#catalog-{{::layertreenodeCtrl.uid}}"
    ng-if="::layertreenodeCtrl.node.children && layertreenodeCtrl.depth < 3">
-  {{::layertreenodeCtrl.node.name}}
+  {{::layertreenodeCtrl.node.name | translate}}
```

Comment:
@pgiraud if you agree with https://github.com/Geoportail-Luxembourg/geoportailv3/commit/1936a35883b0eb323de09b03b3839bc5c9f6dc64 please add it to your branch before merging. Thanks.